integration-tests/rita.py

```diff
@@ -414,33 +558,36 @@ def check_log_contains(f, x):
 
 
 if __name__ == "__main__":
-    a = Node(1, 10)
-    b = Node(2, 25)
-    c = Node(3, 60)
-    d = Node(4, 10)
-    e = Node(5, 0)
-    f = Node(6, 50)
-    g = Node(7, 10)
-    # h = Node(8, 0)
-
+    a1 = Node(1, 10)
```

Comment:
any reason they got changed to letter-number? did it get too confusing lol

Reply:
Yeah, didn't want to wonder which IP a node has every time